### PR TITLE
FINCN-172

### PR DESCRIPTION
**Pull request by maektwain**
Actually this is more like a PR for FINCN-173 and 172. Lets get it reviewed, then I will do the rest of the libs and other FINCN services.

### CODE REVIEW

File: api/build.gradle [hunk 1]
```diff
@@ -1,65 +1 @@
...
publishing {
    publications {
        apiPublication(MavenPublication) {
            from components.java
            groupId project.group
            artifactId project.name
            version project.findProperty('externalVersion') ?: project.version
        }
    }
}
```
Comment (aasaru) [inline api/build.gradle:1]: Something is wrong with newline characters. please check.

**Comment by aasaru** [comment]
Hi @maektwain ! 
Thanks for picking up this topic, it is very important and very valuable to community.

I have created separate branches named "spring_boot_2" to each of the fineract-cn repository.
Our CI and artifact building system is configured to build these artifacts completely separately.

Please send your PR to this branch and (also check build.gradle newline characters):
https://github.com/apache/fineract-cn-anubis/tree/spring_boot_2

Then we can merge your PR and then anubis will be available as a dependency to bracnh spring_boot_2 of other fineract-cn repositories. Take a look to an example here:
https://github.com/apache/fineract-cn-identity/blob/spring_boot_2/shared.gradle#L23

Currently your PR is made to branch develop but if we merge this then everything else breaks until you have managed to send PR-s to dependent repositories. This way (using separate branch spring_boot_2) we can avoid such downtime.

Thanks!

**Review by aasaru — CHANGES_REQUESTED** [review]

**Comment by xurror** [comment]
I'm assisting @maektwain on this. I have forked and made simiar upgrades to the other 10 libraries in FINCN. I'll send the PRs to branch spring_boot_2 as you have indicated

File: api/build.gradle (hunk 1 above)
Comment (maektwain) [reply api/build.gradle:1]: Yes might fix this in the next PR

**Comment by aasaru** [comment]
@xurror . Clear. Let's close this PR then.

**Pull request closed without merging**